### ISSUE TITLE
Import not working: 'The import file is missing the following columns: "name"'

### ISSUE THREAD

**Issue by getwebM** [vendure-ecommerce/vendure#2132]
**Describe the bug**
When trying to import products is always only displayed
Imported 0 items
Done!

You can see the following error message if you have the errors output via a console.log in import-parser.js
[ 'The import file is missing the following columns: "name"' ]
Since my import file has the "name" column and this is also filled with a product, I don't know where the error is.

I have following import.csv:
[import_file_testdaten.csv](https://github.com/vendure-ecommerce/vendure/files/11288229/import_file_testdaten.csv)

I have added the custom fields in vendure-config.ts
`  customFields: {
    Product: [
      { name: 'anwendungshinweise', type: 'string' },
      { name: 'aufbewahrungshinweise', type: 'string' },
      { name: 'aufbewahrung', type: 'string' },
      { name: 'bearbeitungsstand', type: 'string' },
      { name: 'darreichungsform', type: 'string' },
      { name: 'gegenanzeigen', type: 'string' },
      { name: 'monopraeparat', type: 'string' },
      { name: 'nebenwirkungen', type: 'string' },
      { name: 'pflanzlichesArzneitmittel', type: 'string' },
      { name: 'traditionellAngewandtesArzneimittel', type: 'string' },
      { name: 'verschreibungspflicht', type: 'string' },
      { name: 'wichtigeHinweise', type: 'string' },
      { name: 'wirkungsweise', type: 'string' },
      { name: 'hilfsstoffe', type: 'string' },
      { name: 'wirkstoff', type: 'string' },
      { name: 'hersteller', type: 'string' },
      { name: 'packungsgroesse', type: 'string' },
      { name: 'preisProEinheit', type: 'string' },
    ],
  },`

**Expected behavior**
Import the one product from the csv

**Environment (please complete the following information):**
 - @vendure/core version: 1.9.5
 - Nodejs version: 16.10.0
 - Database: 10.5.18-MariaDB-0+deb11u1 - Debian 11

**Comment by getwebM** [comment]
After further testing I found out that it is due to the encoding. My file was no longer UTF-8 encoded by excel. However, I now get a different error message in vendure-import-error.log:
ER_BAD_FIELD_ERROR: yarn de 'customFieldsAnwendungshinweise' in 'field list'

**Comment by michaelbromley** [comment]
Hi,

Have you run a migration after defining your custom fields so that the custom field columns get created in your database? See https://www.vendure.io/docs/developer-guide/migrations/

**Comment by getwebM** [comment]
Yes, I actually forgot about the migration during this run. After carrying out the import as described in the documentation, it worked. Thanks!
